app/assets/stylesheets/pageflow/themes/default/navigation.scss

```diff
@@ -48,13 +48,14 @@ $navigation-opacity: 0.9 !default;
 $navigation-current-page-marker-color: $main-color !default;
 
 @import "./navigation/widget_margin";
+@import "./navigation_share_box";
+@import "./navigation_share_box_icon_colors";
```

Comment:
`navigation_share_box` is a sibling of `navigation` now and should therefore be imported in `default/base.scss` just like `navigation.scss`. `navigation_share_box_icon_colors.scss` could then also be `navigation_share_box/icon_colors.scss`.